homeassistant/components/frontier_silicon/manifest.json

```
@@ -2,7 +2,9 @@
   "domain": "frontier_silicon",
   "name": "Frontier Silicon",
   "codeowners": ["@wlcrs"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/frontier_silicon",
   "iot_class": "local_polling",
-  "requirements": ["afsapi==0.2.7"]
+  "requirements": ["afsapi==0.2.7"],
+  "ssdp": [{ "st": "urn:schemas-frontier-silicon-com:undok:fsapi:1" }]
```

Keep ssdp for follow-up PR
```suggestion
  "requirements": ["afsapi==0.2.7"]
```